cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -321,6 +321,17 @@
     </description>
   </property>
 
+  <property>
+    <name>system.app.config.dir</name>
+    <value>/opt/cdap/master/system-app-config</value>
+    <description>
+      Directory that contains system app config steps that should be executed on
```

Comment:
this isn't really running steps like the bootstrap service. It's not imperative, but declarative. The configs are defining what the desired state is, and CDAP is figuring out how to make that happen. It would be more accurate to say something like:

Directory that contains definitions for what system apps should be running. This directory is continuously monitored for new/modified config files and updating application state to match what is defined in those files.